packages/backend/src/spsp/routes.ts

```diff
@@ -46,18 +45,15 @@ async function getPay(
 
   try {
     const { ilpAddress, sharedSecret } = deps.streamServer.generateCredentials({
```

Comment:
Existential question:
Do `routes.ts` files need `ServiceDependencies` if they can just access things via
```suggestion
    const streamServer = await ctx.container.use('streamServer')
    const { ilpAddress, sharedSecret } = streamServer.generateCredentials({
```

Reply:
I wouldn't mind keeping `ServiceDependencies` for consistency IMO